src/deadline/keyshot_submitter/Submit to AWS Deadline Cloud.py

```diff
@@ -55,9 +55,11 @@ def apply_sticky_settings(self, sticky_settings: dict):
 
         for param in input_parameter_values:
             if param.get("name") and param.get("value"):
-                # don't re-use KeyShotFile param if the render settings are copied from one file to another
-                if param["name"] != "KeyShotFile":
-                    updated_parameter_values[param["name"]] = param["value"]
+                # Don't re-use KeyShotFile param if the render settings are copied from one file to another
+                # Don't preserve conda settings so the Keyshot version can be updated by updating the submitter
+                if param["name"] in ["KeyShotFile", "CondaPackages", "CondaChannels"]:
```

Comment:
Why aren't we using sticky settings for `CondaChannels`? Customers may want to add another Conda channel (e.g. `conda-forge`) in addition to the `deadline-cloud` channel, and the `deadline-cloud` channel name won't change, right?

I also think we could add some extra logic around `CondaPackages` to ensure we have the correct `keyshot` and `keyshot-openjd` versions but retain any additional packages. But that's out of scope for this PR, customers also always have the option to customize the submitter 🙂

Reply:
The use case I had in mind was if someone had a scene in Keyshot 2024 and then updated to Keyshot 2025, their conda packages parameter should adjust to match instead of pulling from the last submission.

If we detected if someone had edited the channels, that would cover both cases but I don't think that's worth the complexity, especially if someone can edit the plugin themselves like you pointed out.

Those are my thoughts but I'm open to changing it.

Reply:
I agree that because customers can always update this themselves, it's not a blocker for this PR and this is a reasonable default.